modules/node/src/channel/channel.service.ts

```diff
@@ -239,6 +239,40 @@ export class ChannelService {
     return txRes;
   }
 
+  async migrateLegacyChannel(userPublicIdentifier: string): Promise<void> {
```

Comment:
how will this fn be called?

will probably want to make this more general, since the "legacy channel" format will depend on store versions we are maintaining vs. store versions the client has (im assuming these store versions are client driven atm)

Reply:
This was a method I started working on before Liam made the migration script, so this isn't needed anymore.